Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright (c) 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+pipeline {
+    agent any
+    options{
+        timestamps()
+        buildDiscarder(logRotator(numToKeepStr: '91'))
+        timeout(time: 12, unit: 'HOURS')
+    }
+    triggers {
+        cron('30 8 * * *')
```

Comment:
@bill-mahoney This should match existing job: https://jenkins.edgexfoundry.org/view/Snap/job/edgex-go-snap-master-stage-snap/configure

should be `@daily`